src/App.js

```diff
@@ -9,40 +9,22 @@ import HomePage from "./pages/HomePage/HomePage";
 import ProductListingPage from "./pages/ProductListingPage/ProductListingPage";
 import Signin from "./pages/Signin/Signin";
 import WishlistPage from "./pages/WishlistPage/WishlistPage";
+import AuthProvider, { AuthContext } from "./AuthProvider";
+import axios from "axios";
```

Comment:
Separate the module import with local import will increase the readability
Ex:
//All module imports at the start
import axios from "axios";

//All local import 
import Signin from "./pages/Signin/Signin";